cdapython/Q.py

```diff
@@ -152,13 +148,18 @@ def __init__(
         self.query: Query = Query()
         self._show_sql: bool = False
         self.dry_run: bool = False
+        self.raw_Q_string = ""
+        self._system = ""
+        self.limit = None
+        self.offset = None
         if len(args) == 1:
             if args[0] is None:
                 raise RuntimeError("Q statement parse error")
 
             if isinstance(args[0], Query):
                 self.query = args[0]
             else:
+                self.raw_Q_string = args[0]
                 query_parsed: Query = where_parser(
                     args[0].strip().replace("\n", " "), debug=debug
                 )
```

Comment:
I don't understand why this requires one or three arguments. And I write the documentation.